config/default.yaml

```
@@ -532,3 +532,6 @@ deployments:
     # the KubernetesPodOperator and KubernetesExecutor. This is what signals to apply the
     # required RoleBindings.
     allowPodLaunching: true
+
+  # Deployment Service account annotation key, depends on cloud provider. Default value for AWS
+  serviceAccountAnnotationKey: eks.amazonaws.com/role-arn
```

```suggestion
  # Annotation name to map the `cloud-role` from creating a deployment to on the Deployment Service account This is the correct value for AWS
  cloudRoleServiceAccountAnnotationName: eks.amazonaws.com/role-arn
```

Slight change of name to make this a bit clearer to understand just based on the config file.